bluebottle/bb_projects/serializers.py

```diff
@@ -207,7 +201,5 @@ class Meta:
                   'organization', 'language', 'account_holder_name',
                   'account_holder_address', 'account_holder_postal_code',
                   'account_holder_city', 'account_holder_country',
-                  'account_iban', 'account_bic', 'account_number',
-                  'account_bank_name', 'account_bank_address',
-                  'account_bank_postal_code', 'account_bank_city',
-                  'account_bank_country', 'account_other', 'tasks')
+                  'account_number', 'account_bic',
+                  'account_bank_country', 'tasks')
```

Comment:
Do we still need the country? Can that be different from account_holder_country?